docs/src/main/asciidoc/security-oidc-bearer-token-authentication.adoc

```diff
@@ -112,6 +113,42 @@ If UserInfo is the source of the roles then set `quarkus.oidc.authentication.use
 
 Additionally, a custom `SecurityIdentityAugmentor` can also be used to add the roles as documented in xref:security-customization.adoc#security-identity-customization[Security Identity Customization].
 
+[[token-scopes-and-security-identity-permissions]]
+=== Token Scopes And SecurityIdentity Permissions
+
+SecurityIdentity permissions are mapped in the form of the `io.quarkus.security.StringPermission` from the scope parameter of the <<token-claims-and-security-identity-roles,source of the roles>>, using the same claim separator.
+
+[source, java]
+----
+import jakarta.inject.Inject;
+import jakarta.ws.rs.GET;
+import jakarta.ws.rs.Path;
+
+import org.eclipse.microprofile.jwt.Claims;
+import org.eclipse.microprofile.jwt.JsonWebToken;
+
+import io.quarkus.security.PermissionsAllowed;
+
+@Path("/service")
+public class ProtectedResource {
+
+    @Inject
+    JsonWebToken accessToken;
+
+    @PermissionsAllowed("email") <1>
```

Comment:
The example is good, though if we used a more permission like value like `orders_read` (or something similar from your original Permissions example) (Curity docs are pretty good covering the various overloaded meanings of scopes - they can represent groups of claims or used for authorization decisions - which was really the original meaning of them from the OAuth2 days, https://curity.io/resources/learn/scopes-vs-claims).

If you prefer to keep the example as is then it is fine, the important message is that users can rely on the auto mapping of the scope claim value to permissions

Reply:
@sberyozkin I don't really have opinion hear, and thanks for the link, I'll read it by the end of the day, but IMHO it's better to use OIDC scope that everyone knows that when it is present, you can expect verified email claim as well thanks to it. So I'd prefer to keep it, but appreciated.

Reply:
my point was it's the standard claim, so people could know what the example is talking about.

Reply:
Yeah, true, it just does not read as something that can be used for authorization control decisions, but it works in the current form as well

Reply:
Read your link and added another resource method. If you prefer changes, shout :-)

Reply:
That said, `email` is good, user has authorized the application to access email

Reply:
@sberyozkin yeah, I kept it :-) and added a new one.